client/packages/common/src/ui/components/inputs/Autocomplete/AutocompleteWithPagination.tsx

```diff
@@ -167,6 +176,18 @@ export function AutocompleteWithPagination<T extends RecordWithId>({
     setTimeout(() => setIsLoading(false), LOADER_HIDE_TIMEOUT);
   }, [options]);
 
+  const getIsOpen = () => {
+    // Use passed in or default if not using android keyboard
+    if (!keyboard.isEnabled) return open;
+
+    // Keep popper closed until keyboard is open
+    // keyboard moves popper up & out of correct position
+    if (!keyboard.isOpen) return false;
+
+    // Use externally controlled `open` state if provided
+    return open ?? keyboardSelectControl.isOn;
+  };
+
```

Comment:
I noticed this pattern has been defined twice so far. If we plan to use it more in the future, I'm wondering if it would be a good idea to define it as a reusable function, either on its own or provided by the context. 😃

Reply:
Yeah great point - thought this could be part of clean up issue somewhere to consolidate the different autocomplete components, but andrei has fixed in his commit! 🔥